modules/ROOT/pages/manage-nodes.adoc

```diff
@@ -35,7 +35,17 @@ The following example shows the environment parameters that are required for a w
 ** `RTF_INSTALLER_IP="10.0.42.86"`
 ** Remove RTF_ACTIVATION_DATA if it is present.
 +
-The following example shows the environment parameters that are required for a dedicated node. If you copied the environment file from a controller node, delete all lines in the file except those shown in the example. Use the values in the file you copied for `RTF_DOCKER_DEVICE_SIZE` and `RTF_TOKEN`. For `RTF_INSTALLER_IP`, specify the IP address of the lead controller node.
+The following example shows the environment parameters that are required for a controller node. Use the values in the file you copied for `RTF_DOCKER_DEVICE_SIZE`, `RTF_TOKEN`, and `RTF_ETCD_DEVICE`. Use the IP address of the lead controller node for `RTF_INSTALLER_IP`.
```

Comment:
"Use the values in" seems ambiguous to me. "In" could mean that the user should make use of the values within the file, or it could mean that the use should make use of the values that the file contains.
How about "Use the values from"?
Same goes for "Use the values in" at line 48.